tests/test_email_service.py

```
@@ -0,0 +1,338 @@
+            }
+        )
+        
+        email3 = await email_service.create_email(db_session, request_different_caller)
+        
+        # Should create a new email for different caller
+        assert email3.id != email1.id
+        assert email3.caller_id == "service-b"
+
+
+@pytest.mark.asyncio
+async def test_caller_id_is_required(db_session):
```

Ruff (0.14.10)

Unused function argument: `db_session`

(ARG001)